pcsx2/MTGS.cpp

```diff
@@ -250,7 +250,7 @@ void SysMtgsThread::OpenGS()
 	GSsetBaseMem(RingBuffer.Regs);
 	GSirqCallback(dummyIrqCallback);
 
-	pxAssertMsg((GSopen2((void**)pDsp, 1 | (renderswitch ? 4 : 0)) == 0), "GS failed to open!");
+	pxAssertMsg((GSopen2(g_gs_window_info, 1 | (renderswitch ? 4 : 0)) == 0), "GS failed to open!");
```

Comment:
I'm not a fan of storing this in a global, but I couldn't see any obvious way to get it through to MTGS/SysCoreThread. Ideally, this would be computed on demand by the host, and passed through on initialization.